app/scripts/views/components/shareVirtualStudy/shareVirtualStudy.js

```diff
@@ -154,18 +154,18 @@
                     if (saveCohort) {
                       var _selectedSamplesNum = 0;
                       var _selectedPatientsNum = 0;
-                      _.each(self_.stats.selectedCases, function (studyCasesMap) {
+                      _.each(self_.stats.studies, function (studyCasesMap) {
                         _selectedSamplesNum += studyCasesMap.samples.length;
-                        _selectedPatientsNum += studyCasesMap.patients.length;
                       });
                       self_.selectedSamplesNum = _selectedSamplesNum;
-                      self_.selectedPatientsNum = _selectedPatientsNum;
+                      self_.selectedPatientsNum = _selectedSamplesNum;
```

Comment:
# of patients always matches with # of samples?

Reply:
Sorry! by mistake changed it.
 btw where are `selectedSamplesNum` and `selectedPatientsNum` used here?